mne/io/bti/bti.py

```diff
@@ -1060,42 +1016,57 @@ def __init__(self, pdf_fname, config_fname='config',
                  translation=(0.0, 0.02, 0.11), convert=True,
                  rename_channels=True, sort_by_ch_name=True,
                  ecg_ch='E31', eog_ch=('E63', 'E64'),
-                 verbose=None):
-
-        if pdf_fname is None:
-            raise ValueError('pdf_fname must be a path, not None')
+                 preload=None, verbose=None):
+        if preload is None:
+            warnings.warn('preload is True by default but will be changed to '
+                          'False in v0.12. Please explicitly set preload.',
+                          DeprecationWarning)
```

Comment:
Let's tackle that in a separate issue. What's here is our standard procedure, so let's figure out what's going on with your setup separately.